src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -66,10 +65,10 @@ public static void Run(AddSourceArgs args, Func<ILogger> getLogger)
 
             if (newPackageSource.IsHttp && !newPackageSource.IsHttps && !newPackageSource.AllowInsecureConnections)
             {
-                getLogger().LogWarning(
+                throw new ArgumentException(
                     string.Format(CultureInfo.CurrentCulture,
-                        Strings.Warning_HttpServerUsage,
-                        "add source",
+                        Strings.Error_HttpSource_Single_Short,
+                        "source add",
```

Comment:
The command name in docs is [dotnet nuget add source](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-add-source), hence I assume `add source` is correct. Is there any reason why we changed the string from `add source` to `source add`?

Reply:
Oh i was looking at nuget.exe which has it as "source add". I will reverse since both can be valid